primer/brand
# Backfilling / updating design tokens and button size correction

## Summary

Part of https://github.com/github/primer/issues/1487

## List of notable changes:



- **added** new shadow tokens for secondary button states
- **updated** button heights to match Figma. Reduced height across the board. All affected components have snapshots updated. 
- **removed** unused tokens, including all `brand.color-accent-*` in favour of a reduced set

## What should reviewers focus on?

- Interact with buttons and make sure the shadows work as expected
- Verify height of buttons match Figma

## Steps to test:



1. Check the design token diff in CI to verify changes
1. Verify that button height matches Figma

## Supporting resources (related issues, external links, etc):

- [Figma spec](https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=1020%3A29690&t=GpFm0cWncH8Zq1qD-4)

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

### Medium button, which should be 48px in height
<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

![before](https://user-images.githubusercontent.com/13340707/207316060-a4854a24-63d8-47a7-9390-6e39caba2db4.png)

 </td>
<td valign="top">

![after](https://user-images.githubusercontent.com/13340707/207316078-c6f5aeb9-cc9f-4be3-9aba-599ef100e52c.png)


</td>
</tr>
</table>


### Large button, which should be 56px in height
<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

![before](https://user-images.githubusercontent.com/13340707/207316667-f2bf7942-f8e7-4ab0-b0da-0a8c33474aaa.png)

 </td>
<td valign="top">

![after](https://user-images.githubusercontent.com/13340707/207316678-47832e16-6dad-4ec5-9a6f-80f39c1e1f96.png)

</td>
</tr>
</table>
